parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -111,19 +111,15 @@ installStandaloneContainerd() {
         removeMoby
         removeContainerd
         updateAptWithMicrosoftPkg
-        # TODO: first try downloading from microsoft apt repo then fallback to our storage ep
-        downloadContainerd ${CONTAINERD_VERSION}
-        wait_for_apt_locks
-        retrycmd_if_failure 10 5 600 apt-get -y -f install ${CONTAINERD_DEB_FILE} || exit $ERR_CONTAINERD_INSTALL_TIMEOUT
-        rm -Rf $CONTAINERD_DOWNLOADS_DIR &
+        apt_get_install 20 30 120 moby-containerd=${CONTAINERD_VERSION}* --allow-downgrades || exit $ERR_CONTAINERD_INSTALL_TIMEOUT
```

Comment:
all these unneccessary changes will be fixed after I consolidate our install_containerd logic to check on packages.microsoft first before checking on the storage ep.

for now we'll need to go back and forth between the two depending on the version.

Reply:
we should really get this fixed sooner.